src/databricks/labs/remorph/install.py

```diff
@@ -134,38 +198,158 @@ def warehouse_type(_):
     @retried(on=[NotFound], timeout=timedelta(minutes=5))
     def setup_catalog(self, catalog_name: str):
         allow_catalog_creation = self._prompts.confirm(
-            f"""Catalog {catalog_name} not found.\
+            f"""Catalog `{catalog_name}` not found.\
                     \nDo you want to create a new one?"""
         )
         if not allow_catalog_creation:
             msg = "Catalog is needed to setup Remorph"
             raise SystemExit(msg)
 
-        logger.info(f" Creating new Catalog {catalog_name}")
+        logger.info(f" Creating new Catalog `{catalog_name}`")
         self._catalog_setup.create(catalog_name)
 
     @retried(on=[NotFound], timeout=timedelta(minutes=5))
     def setup_schema(self, catalog_name: str, schema_name: str):
         allow_schema_creation = self._prompts.confirm(
-            f"""Schema {schema_name} not found in Catalog {catalog_name}\
+            f"""Schema `{schema_name}` not found in Catalog `{catalog_name}`\
                     \nDo you want to create a new Schema?"""
         )
         if not allow_schema_creation:
             msg = "Schema is needed to setup Remorph"
             raise SystemExit(msg)
 
-        logger.info(f" Creating new Schema {catalog_name}.{schema_name}")
+        logger.info(f" Creating new Schema `{catalog_name}.{schema_name}`")
         self._catalog_setup.create_schema(schema_name, catalog_name)
 
+    def prompt_for_new_installation(self):
+        """
+        Prompts for the new Installation and returns the configuration
+        :return: MorphConfig
+        """
+        module_prompt = self._prompts.choice(
+            "Which module(s) would you like to configure:", ["transpile", "reconcile", "all"]
+        )
+
+        morph_config, reconcile_config = None, None
+        match module_prompt:
+            case "transpile":
+                morph_config = self._prompt_for_transpile_setup()
+            case "reconcile":
+                reconcile_config = self._prompt_for_reconcile_setup()
+            case "all":
+                morph_config, reconcile_config = self._prompt_for_all()
+
+        return morph_config, reconcile_config
+
+    def _prompt_for_all(self) -> tuple[MorphConfig, ReconcileConfig]:
+        return self._prompt_for_transpile_setup(), self._prompt_for_reconcile_setup()
+
+    def _prompt_for_transpile_setup(self) -> MorphConfig:
+        logger.info("\nPlease answer a few questions to configure remorph: ** transpile **")
+
+        # default params
+        catalog_name = "transpiler_test"
+        schema_name = "convertor_test"
+        ws_config = None
+
+        source = self._prompts.choice("Select the source:", list(SQLGLOT_DIALECTS.keys()))
+
+        input_sql = self._prompts.question("Enter Input SQL path (directory/file)")
```

Comment:
would be good to validate if this file exists, just in case

Reply:
We will take care of the `Validation` while running the `CLI commands`.